.github/workflows/publish.yaml

```diff
@@ -0,0 +1,51 @@
+name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+
+on: push
```

Comment:
IIUC, this workflow runs on every push, but we are only publishing to PyPi when there's a new tag. Is there another reason to run it on every push besides testing the build process? If not, we could restrict it to only run on tags:
```
on:
  push:
    tags:
       - "*"

Reply:
Hmm, i agree,  to avoid invalidating your current approval, I'd like to go ahead and  merge.